odl/discr/lp_discr.py

```diff
@@ -605,6 +695,108 @@ def show(self, method='', title=None, indices=None, show=False, fig=None,
                                   show=show, fig=fig, **kwargs)
 
 
+def uniform_discr_frompartition(partition, exponent=2.0, interp='nearest',
```

Comment:
The lack of this became obvious in this branch, so that's how it ended up here. I can cherry-pick the commit to master if that helps.